PayKids/presentation/src/main/java/com/paykids/presentation/view/diary/DiaryFragment.kt

```
@@ -0,0 +1,8 @@
+package com.paykids.presentation.view.diary
+
+import androidx.fragment.app.Fragment
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class DiaryFragment: Fragment() {
+}
```

detekt (1.23.7)

[warning] The class or object DiaryFragment is empty.

(detekt.empty-blocks.EmptyClassBlock)